scanner/src/test/java/ca/on/oicr/gsi/runscanner/scanner/processor/IlluminaProcessorTest.java

```diff
@@ -23,8 +23,8 @@ protected NotificationDto process(File directory) throws IOException {
 
   @Override
   public void testGoldens() throws IOException {
-    if (!System.getProperty("skipIllumina", "true").equals("true")) {
-      checkDirectory("/illumina");
-    }
+    // if (!System.getProperty("skipIllumina", "true").equals("true")) {
```

Comment:
Looks like this might have stayed commented out by accident?

Reply:
Couldn't get this setting to work in intellij, will uncomment these lines once the code is approved.